### PR TITLE
Add pydeck

### PR DESCRIPTION
**Pull request by ajduberstein**
Adds recipe for [pydeck](https://pypi.org/project/pydeck/) to conda-forge.

The pydeck library for high-scale data visualization, backed by deck.gl

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pydeck/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d58bc0a08707a613ccf1aa99ccdab3b040ac6d93c70046ef165cae913b527fd8"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (marcelotrevisani) [inline recipes/pydeck/meta.yaml]: I think this recipe can be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```

**Comment by ajduberstein** [comment]
@marcelotrevisani it seems that `noarch` causes a build failure–let me know if you're fine with leaving it out.

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani it seems that `noarch` causes a build failure–let me know if you're fine with leaving it out.

The failure was not related to the ``noarch``, could you please return to the ``noarch``?

**Comment by ajduberstein** [comment]
Updated, @marcelotrevisani – let me know if I'm able to merge

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**